Gemfile.lock

```diff
@@ -187,4 +189,4 @@ RUBY VERSION
    ruby 2.5.3p105
 
 BUNDLED WITH
-   1.17.3
+   2.0.1
```

Comment:
I think we want to avoid bundler 2 for now, as it's backwards incompatible. So I think we may need to regenerate this file.

```
gem uninstall bundler      # pick the 2.0.1 one if you have options
gem install bundler -v 1.17.3
rm Gemfile.lock
bundle install
```

Alternatively, you could do this in Docker, with `docker-compose run web bundle install`.